test/features/auth.feature

```diff
@@ -0,0 +1,97 @@
+@auth
+Feature: auth module
+
+    Scenario: get list of all tokens in invalid keyspace
+        Given authorisation
+        When make a GET request to "/auth/token/invalid-keyspace"
+        Then the response status code should be 200
+        And  the response should contain:
+            | clients | [] |
+
+    Scenario: get list of all tokens in my keyspace
+        Given authorisation
+        And make a POST request to "/auth/token" with:
+            | serverId | "eddiehub"    |
+            | scopes   | ["Data.Read"] |
+        When make a GET request to "/auth/token/eddiehub"
+        Then the response status code should be 200
+        And  the response property "clients" should be of type "TYPE:UUID"
```

Comment:
Would this not be a collection of `UUID`s?

Reply:
In theory yes, but I return them [as an Array](https://github.com/EddieHubCommunity/api/pull/171/files#diff-aea62a4c305374791411369a46e8553d716a032a03bb48136176873887e3311cR91) because key and value are the same

Reply:
Oh I see now, sorry I did not check the step def. The step def checks for each item in the array, I read the feature test as it was only 1 item.

Maybe it could be renamed to something like...
```suggestion
        And the response property "clients" should have a collection of type "TYPE:UUID"
```